templates/staff/user_list.html

```diff
@@ -24,6 +24,12 @@
 <div class="jumbotron jumbotron-fluid jumbotron--danger pt-md-2">
   <div class="container">
     <h1 class="display-4">Users</h1>
+
+    <div class="d-flex">
+      <a class="btn btn-primary" href="{% url 'staff:user-create' %}">
+        Create a User
```

Comment:
Could this be called "Create Interactive User", please? It caused me some confusion at first.